PodcastGenerator/admin/navbar.php

```diff
@@ -59,9 +59,12 @@
                 </div>
             </li>
         </ul>
-        <ul class="navbar-nav ml-auto">
-            <li class="nav-item nav-pills">
-                <a class="nav-link active p-1" href="logout.php">Logout - <?php echo $_SESSION["username"]; ?></a>
+        <ul class="navbar-nav">
+            <li class="nav-item">
+                <a class="nav-link" href="<?php echo $config['url']; ?>" target="_blank"><?php echo _('View Podcast'); ?></a>
```

Comment:
Is this intended to go to the main podcast page?

Reply:
I believe yes... it is inside the admin area..